src/plugins/undoRedo/__tests__/UndoRedo.spec.js

```diff
@@ -2952,6 +2952,89 @@ describe('UndoRedo', () => {
       }, 100);
     });
 
+    it('should fire a `beforeUndoStackChange` and `afterUndoStackChange` hooks after ' +
```

Comment:
What do you think about moving the hook test suites to a separate file? For example `hooks.spec.js`? The main file is big enough and I think that it deserves slimming therapy 😅 

You've added tests that check if the hooks are triggered but I don't see tests that check if the arguments are passed correctly to them. Can I ask you to complete the tests?

Reply:
> You've added tests that check if the hooks are triggered but I don't see tests that check if the arguments are passed correctly to them. Can I ask you to complete the tests?

Changed within: https://github.com/handsontable/handsontable/pull/7852.